README.md

```diff
@@ -0,0 +1,30 @@
+﻿# Fasten90
+
+Fasten90 is a project to create a prototype of a tool to use reportlab to export md and csv data to a PDF document. 
+
+## Installation
+
+Use the install.bat to install Fasten90.
+
+```python
+python -m venv venv
+
+venv\Scripts\python.exe -m pip install -r requirements.txt
+```
+
+## Usage
+
+Under development.
+```
```

Comment:
Can you finish or remove this 'code block' ?

The content at below shall not put into code block.